core/misc/tabledrag.js

```diff
@@ -206,7 +206,7 @@ Backdrop.tableDrag.prototype.addColspanClass = function(columnIndex) {
     });
     if (index > 0) {
       cell = cells.filter(':nth-child(' + index + ')');
-      if (cell[0].colSpan && cell[0].colSpan > 1) {
+      if (cell[0] != null && cell[0] != undefined && cell[0].colSpan && cell[0].colSpan > 1) {
```

Comment:
Since you are using loose comparison, there is no need to check for both. `null` and `undefined` are "same" when using loose comparison (including `!=`). You can see this by running `console.log(null == underfined);` which returns true.

This is an aside from whether this is needed or not. I'm not sure I understand why you are not getting a value on `cell[0]`, since that cell was expressly selected in the `each()` above, so it should exist. Is your table malformed?